crates/examples/data-generators/image-processing-generator/README.md

```diff
@@ -0,0 +1,2 @@
+We use the image [Veracruz Puerto - Vista desde el Hotel Emporio](veracruz.jpg) by Eduardo Pavon as a test dataset for some of our examples.
+This image is licensed under the [Creative Commons Attribution Share Alike 2.0 Generic (CC-BY-SA 2.0) license](LICENSE_CC-BY-SA-2.0.md).
```

Comment:
A newline at the end of this file would be good.

Reply:
Done